install_config/aggregate_logging.adoc

```diff
@@ -2054,6 +2054,244 @@ $ oc exec <$espod> -- es_load_kibana_ui_objects <user-name>
 
 Where `$espod` is the name of any one of your Elasticsearch pods.
 
+[[kibana-visualizations-custom]]
+*Adding Custom Fields to Kibana Visualize*
+
+If your {product-title} cluster is configured with custom logging fields
```

Comment:
What about `If your {product-title} cluster generates logs in json format, which consist of custom fields` ?